freqtrade/rpc/fiat_convert.py

```diff
@@ -86,7 +86,7 @@ def _get_gekko_id(self, crypto_symbol):
                     return None
             else:
                 return None
-        found = [x for x in self._coinlistings if x['symbol'] == crypto_symbol]
+        found = [x for x in self._coinlistings if x['symbol'].lower() == crypto_symbol]
```

Comment:
It's surely not wrong to do this, but it can/will have unintended side-effects.

My fear with this is that it'll make the "duplicates - can't convert" problem worse (where we need an explicit mapping).

there's `13579` unique id's, with `11280` unique "symbol" descriptors - and `11140` if the casing is ignored.
This means, we've now 140 more collisions in these coins - where it's impossible to reliably determine which one is the correct one (it's usually obvious once you look at it - but not reliably done via code).

Now most impacted (looking at your list now) don't look tradeable at first glance - but not all ...